evmrpc/filter.go

```diff
@@ -23,6 +25,8 @@ import (
 
 const TxSearchPerPage = 10
 
+const MaxNumOfWorkers = 500
```

Comment:
nit do we want to allow this to be tunable for quicknode? i.e. shall we make it a config?

Reply:
Yeah we definitely could, but I feel this config doesn't need to be tuned much, max concurrency of 500 is already a lot, adding more goroutines probably wouldn't help any futher more